courses/models.py

```diff
@@ -14,6 +14,7 @@ class Program(models.Model):
     title = models.CharField(max_length=255)
     live = models.BooleanField(default=False)
     description = models.TextField(blank=True, null=True)
+    personalized_pricing = models.BooleanField(default=False)
```

Comment:
To be clear, are you talking about `financial_aid_availability`? I went with `personalized_pricing` because it's the name on the ticket, but I'm not picky about naming so if you'd prefer to stick with what you've got that's fine by me.